src/content/learn/passing-props-to-a-component.md

```diff
@@ -180,19 +180,19 @@ function Avatar(props) {
 }
 ```
 
-Usually you don't need the whole `props` object itself, so you destructure it into individual props.
+Зазвичай вам не потрібен весь об'єкт `props` сам по собі, тому ви розкладаєте його на окремі пропси.
 
 <Pitfall>
 
-**Don't miss the pair of `{` and `}` curlies** inside of `(` and `)` when declaring props:
+**Не пропустіть пару фігурних дужок `{` та `}` всередині `(` та `)` при оголошенні пропсів:
 
 ```js
 function Avatar({ person, size }) {
   // ...
 }
 ```
 
-This syntax is called ["destructuring"](https://developer.mozilla.org/docs/Web/JavaScript/Reference/Operators/Destructuring_assignment#Unpacking_fields_from_objects_passed_as_a_function_parameter) and is equivalent to reading properties from a function parameter:
+Цей синтаксис називається ["деструктуризація"](https://developer.mozilla.org/docs/Web/JavaScript/Reference/Operators/Destructuring_assignment#Unpacking_fields_from_objects_passed_as_a_function_parameter) і еквівалентний читанню властивостей з параметра функції:
```

Comment:
```suggestion
Цей синтаксис називається ["деструктуризація"](https://developer.mozilla.org/docs/Web/JavaScript/Reference/Operators/Destructuring_assignment#Unpacking_fields_from_objects_passed_as_a_function_parameter) і є еквівалентним читанню властивостей із параметра функції:
```

Reply:
```suggestion
Цей синтаксис називається ["деструктуризацією"](https://developer.mozilla.org/docs/Web/JavaScript/Reference/Operators/Destructuring_assignment#Unpacking_fields_from_objects_passed_as_a_function_parameter) і еквівалентний витягуванню властивостей з параметра функції:
```

What do you think of this option? It's a bit paraphrased, but in my opinion, it's more understandable

Reply:
@ltlaitoff it's up to you :) I don't mind personal approach to translation